lib/logstash/outputs/elasticsearch_java.rb

```diff
@@ -8,11 +8,14 @@
 require "socket" # for Socket.gethostname
 require "thread" # for safe queueing
 require "uri" # for escaping user input
-require 'logstash-output-elasticsearch_jars.rb'
-
-# This output lets you store logs in Elasticsearch and is the most recommended
-# output for Logstash. If you plan on using the Kibana web interface, you'll
-# need to use this output.
+require "logstash/outputs/elasticsearch_java/protocol"
+
+# This output lets you store logs in Elasticsearch using the native 'node' and 'transport'
+# protocols. It is highly recommended to use the regular 'logstash-output-elasticsearch' output
+# which uses HTTP instead. This output is, in-fact, sometimes slower, and never faster than that one.
+# Additionally, upgrading your Elasticsearch cluster may require you to simultaneously update this
+# plugin for any protocol level changes. The HTTP client has far fewer of these issues and is
+# generally just easier to work with.
```

Comment:
"is generally just easier"

Maybe rephrase "may be easier to work with due to wider familiarity with http" ?